pygeos/strtree.py

```diff
@@ -138,7 +141,8 @@ def query_bulk(self, geometry, predicate=None):
         geometry : Geometry or array_like
             Input geometries to query the tree.  The envelope of each geometry
             is automatically calculated for querying the tree.
-        predicate : {None, 'intersects', 'within', 'contains', 'overlaps', 'crosses', 'touches'}, optional
+        predicate : {None, 'intersects', 'within', 'contains', 'overlaps', 'crosses',\
+            'touches', 'covers', 'covered_by', 'contains_properly'}, optional
```

Comment:
```suggestion
 'touches', 'covers', 'covered_by', 'contains_properly'}, optional
```

(it's a bit ugly in the code, but that's what is needed to not get empty space in the actual docstring